tests/include/sqlpp17_test/compare.h

```diff
@@ -28,15 +28,21 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 #include <cfloat>
 #include <iostream>
+#include <iomanip>
 #include <sstream>
 #include <stdexcept>
 
 namespace sqlpp::test
 {
+  std::string to_decimal_string(float value, int precision)
+  {
+    return static_cast<std::ostringstream&>(std::ostringstream{} << std::setprecision(precision) << value).str();
```

Comment:
I would prefer to avoid the `static_cast`:

```c++
std::ostringstream os;
os << std::setprecision(precision) << value).str();
return os.str();
```

Reply:
That was my initial implementation, the cast just seemed closer to the spirit of the original code.

Fix commited.